connector_phone_palitto_event_manager/controllers/main.py

```diff
@@ -0,0 +1,68 @@
+from odoo import http
+from odoo.http import request
+
+
+class PCS_VOIP(Home):
+
+    def create_cdr_record(self, **kw):
+        vals = {'guid' : kw.get('GUID'),
+                'inbound_flag' : kw.get('Inbound'),
+                'call_start_time' : kw.get('StartTime'),
+                'caller_id' : kw.get('CallerID'),
+                'caller_id_name' : kw.get('CalledIDName'),
+                'state' : 'offering',
+        }
+        request.env['phone.cdr'].create(vals)
+        return True
+
+    @http.route('/web/incomingCall', type='http', auth='public', website=True, sitemap=False)
```

Comment:
Please change the routes to be under a a specific name, `/palitto`, and have the endpoint names match the ones used in the SQL examples in the PDF documentation.
For example, for "Incoming Call", use route `/palitto/incomingCall`